docs/release_process_quarterly.md

```diff
@@ -0,0 +1,501 @@
+
+### 3. Project Reference Updates & Release Creation
+    - [ ] Update version number in pom.xml if not already done
+    - [ ] Merge ‘release/(year)-(quarter)’ branch into ‘master’ branch for the jpo-sdw-depositor project
+    - [ ] Create a release for the jpo-sdw-depositor project from the ‘master’ branch and tag the release with the version number of the release. (e.g. jpo-sdw-depositor-x.x.x)
+    - [ ] Create docker image
+        - [ ] Use the release for the jpo-sdw-depositor to produce docker image with the same tag name, containing the version number.
+        - [ ] Upload docker image to [DockerHub](https://hub.docker.com/u/usdotjpoode)
+        - [ ] Tag docker image with the version number of the app. (e.g. 1.0.0)
+        - [ ] Tag docker image with year and quarter of release. (e.g. 2024-Q2)
+        - [ ] Tag docker image with 'latest' tag for the most recent release.
+    - [ ] Merge master branch into develop branch & verify that CI/CD passes.
+
+### 4. DockerHub Image Testing
+    - [ ] image starts up correctly
+    - [ ] messages are consumed successfully
+    - [ ] messages are submitted to the SDX successfully
+
+
+## jpo-utils
+### Prerequisites
+None
+
+### 1. Code Ready & Release Notes
+    - [ ] Release notes drafted & added to `Release_notes.md` file in `docs` directory
+    - [ ] Code changes for release are merged into `develop`
+    - [ ] A new branch `release/(year)-(quarter)` is created from `develop`
+
+### 2. Preliminary Testing
+    - [ ] jpo-jikkou docker image builds
+    - [ ] jpo-kafka-connect image builds
+        - [ ] Verify all connectors are created using connect rest api
+
+### 3. Project Reference Updates & Release Creation
+    - [ ] Update version number in pom.xml files for the 'jpo-utils' project if not already done.
+    - [ ] Create a release for the jpo-utils project from the ‘master’ branch and tag the release with the version number of the release. (e.g. jpo-utils-x.x.x)
+    - [ ] Merge master branch into develop branch & verify that CI/CD passes.
+
+### 4. DockerHub Image Testing
+    - [ ] Change the image references in `docker-compose-kafka.yml` and `docker-compose-connect.yml` to the version number of the release. (e.g. 1.0.0)
+        - [ ] `docker-compose-kafka.yml` services changed:
+            - [ ] kafka-setup image: `usdotjpoode/jpo-jikkou:1.0.0`
+        - [ ] `docker-compose-connect.yml` services changed:
+            - [ ] kafka-connect image: `usdotjpoode/jpo-kafka-connect:1.0.0`
+            - [ ] kafka-connect-setup image: `usdotjpoode/jpo-jikkou:1.0.0`
+    - [ ] jpo-jikkou docker image runs successfully
+    - [ ] jpo-kafka-connect image runs successfully
+        - [ ] Verify all connectors are created using connect rest api
+
+## jpo-ode
+### Prerequisites
+    - [ ] asn1_codec release completed
+    - [ ] jpo-cvdp release completed
+    - [ ] jpo-security-svcs release completed
+    - [ ] jpo-sdw-depositor release completed
+    - [ ] jpo-utils release completed
```

Comment:
I would be useful to note the nature of each of the dependencies for each project.  Whether the dependency is:
* Submodule
* Java library (Maven), or
* Dockerhub container

So for this, something like:
    - [ ] asn1_codec (submodule) release completed
    - [ ] jpo-cvdp (submodule) release completed
    - [ ] jpo-security-svcs (submodule) release completed
    - [ ] jpo-sdw-depositor (submodule) release completed
    - [ ] jpo-utils (submodule) release completed

Reply:
Addressed!